src/libumf.def.in

```diff
@@ -34,7 +34,8 @@ EXPORTS
     umfMemoryProviderPurgeLazy
     umfMemoryProviderPutIPCHandle
     umfMemspaceDestroy
-    umfOpenIPCHandle
+    umfOpenIPCHandle    
+    umfOsMemoryProviderOps
```

Comment:
`umfOsMemoryProviderOps` is listed twice now

Reply:
done